keyboards/vision_division/keymaps/default/rules.mk

```diff
@@ -1 +1 @@
-AUDIO_ENABLE = yes
+AUDIO_ENABLE = no           # Audio output
```

Comment:
You can actually delete this file; `keyboards/vision_division/rules.mk` will disable Audio for the keyboard, unless someone explicitly sets `AUDIO_ENABLE = yes`.

Reply:
ok, will do - thanks for the review!